README.md

```diff
@@ -8,18 +8,25 @@
 ![pytest](https://github.com/skypilot-org/skypilot/actions/workflows/pytest.yml/badge.svg)
 [![Documentation Status](https://readthedocs.org/projects/skypilot/badge/?version=latest)](https://skypilot.readthedocs.io/en/latest/?badge=latest)
 
-SkyPilot is a framework for easily running machine learning[^1] workloads on any cloud through a unified interface. No knowledge of cloud offerings is required or expected – you simply define the workload and its resource requirements, and SkyPilot will automatically execute it on AWS, Google Cloud Platform or Microsoft Azure.
+SkyPilot is a framework for easily running machine learning workloads[^1] on any cloud. 
 
 
-### Key features
+Use the clouds **easily** and with **low cost**, all without needing to learn cloud infra.
+
+
+
+_Ease of use & productivity_
 * **Run existing projects on the cloud** with zero code changes
-* **No cloud lock-in** – seamlessly run your code across different cloud providers (AWS, Azure or GCP)
-* **Minimize costs** by leveraging spot instances and automatically stopping idle clusters
-* **Automatic recovery from spot instance failures**
-* **Automatic fail-over** to find resources across regions and clouds
+* **Easily manage jobs** across multiple clusters
+* **Automatic fail-over** to find scarce resources (GPUs) across regions and clouds
 * **Store datasets on the cloud** and access them like you would on a local file system 
-* **Easily manage job queues** across multiple clusters
+* **No cloud lock-in** – seamlessly run your code across different cloud providers (AWS, Azure or GCP)
 
+_Cost saving_
+* Run jobs on **spot instances** with **automatic recovery** from preemptions
+* Hands-free life cycle: **automatically stop idle clusters**
+* One-click use of **TPUs**, for high-performance, cost-effective training
+* Automatically benchmark and find the best hardware for your job
```

Comment:
If we're putting this under cost saving, maybe better to say:
* Automatically benchmark and find cost-effective hardware for your job

Reply:
Changed to "cheapest" now, to avoid "cost effective" showing up in 2 successive items.